docs/jwst/photom/main.rst

```diff
@@ -39,17 +42,26 @@ If the photom step finds that the wavelength and relative response arrays are
 populated in the selected table row, it copies those arrays to a table extension
 called "RELSENS" in the science data product.
 
-Finally, for imaging modes, the photom step loads data from a pixel area map
-reference file are also loaded into the science data product. The 2D
+None of the conversion factors are actually applied to the data for these
+observing modes. They are simply attached to the science product.
+
+Pixel Area Data
+^^^^^^^^^^^^^^^
+
+For imaging modes, the photom step loads data from a pixel area map
+reference file and appends it to the science data product. The 2D
 data array from the pixel area map is copied into an image extension
-called "AREA" in the science data product. The values of the PIXAR_SR and
-PIXAR_A2 keywords in the photom reference table are also
-copied into keywords of the same name in the primary header of
-the science data product.
+called "AREA" in the science data product.
 
-Note that for all of these exposure modes the pixel values of the science data
-product are not changed by the photom step. All of the reference data
-are simply loaded into the science data product in some way.
+The process of attaching the pixel
+area data also populates the keywords PIXAR_SR and PIXAR_A2 in the primary
+header of the science product, which give the average pixel area in units of
+steradians and square seconds, respectively.
```

Comment:
Just double checking, this is square seconds and not square arcseconds?

Reply:
Whoops, you're right. It's square arcseconds.